lib/idris-controller.coffee

```diff
@@ -46,6 +47,9 @@ class IdrisController
       @messages.attach()
       @messages.hide()
 
+  stopCompiler: =>
+    @model?.stop()
```

Comment:
There is sonme info in the statusbar, but I'm thinking of removing it, because I have the feeling that there is already a bit too much information there.
The new version shows if a file has been loaded in a small pane below the code. Maybe it's better to unify all the idris messages in such a pane.